forte/processors/tests/nltk_processors_test.py

```diff
@@ -19,6 +19,7 @@ def setUp(self):
         self.nltk = Pipeline()
         self.nltk.set_reader(StringReader())
         self.nltk.add_processor(NLTKSentenceSegmenter())
+        self.nltk.initialize()
```

Comment:
Duplicated line. Should delete it.

Reply:
Okay. Thanks for letting me know. I will fix it in another PR.